### DISCUSSION TITLE
How to get higher quality of MP3 than 128kbps

### DISCUSSION THREAD

**Discussion by J4cker14** [Tyrrrz/YoutubeDownloader#547] [Q&A]
Hi, amazing work, but seems configuraiton option for setting audio bitrate is missing. Am I missing something ?

**Comment by Tyrrrz** [comment]
The method we're currently using to extract streams is limited to 128kbps for audio.

**Reply by J4cker14** [reply]
OK, what is the gap? Reverse engineering of the YT flow, or portation layer only?

**Comment by avengerx** [comment]
I'm also interested in this. I see downloading audio only (mp4 format) from an opus/251 (vbr 0-160kbps) video (from "stats for nerds), it gets converted into `AAC/LC 128kb/s, 44100Hz`. Mp3 also cbr 128kbps, although it "superscales" to 48000Hz

It could at least convert to mp3 vbr to keep closer to what's in the source. All this client does is specify a "quality profile" and the container (mp3/mp4/ogg). Looks like all magic is done in tyrrrz/YoutubeExplode. In a quick look there I couldn't find where it "hardcodes" these quality variables.
